.github/workflows/validate_jsonschemas.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Validate JSON Schemas
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Check out repo
+      uses: actions/checkout@v2
+    - name: Setup Node for gen/postgen scripts
+      uses: actions/setup-node@v2
+      with:
+        node-version: '12'
+    - name: Install JSON schema schema
+      run: |
+        curl -o jsonschema-schema.json http://json-schema.org/draft-04/schema
+    - name: Validate every schema
+      run: |
+        JSON_SCHEMAS=$(find ./jsonschema/google/events -name "*.json")
+        while IFS= read -r line ; do
+          npx ajv-cli@v3.0.0 validate -s jsonschema-schema.json -d $line;
+        done <<< "$JSON_SCHEMAS"
+    - name: Cleanup
```

Comment:
Do we particularly need to clean this up? Can't we regard it as a build artifact like the download of protoc?
Maybe put it in a `tmp` directory to make that clearer?

Reply:
Moved script to `tmp`, so it's ignored naturally. Thanks!